rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -13,6 +13,7 @@ Server runtime for Smithy Rust Server Framework.
 publish = true
 
 [features]
+aws-lambda = ["dep:lambda_http"]
```

Comment:
See https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies for documentation on this syntax